packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -421,6 +425,19 @@ export function attemptUserBlockingHydration(fiber: Fiber): void {
   markRetryTimeIfNotHydrated(fiber, expTime);
 }
 
+export function attemptContinuousHydration(fiber: Fiber): void {
+  if (fiber.tag !== SuspenseComponent) {
+    // We ignore HostRoots here because we can't increase
```

Comment:
~~This comment seems to have been copy-pasted from above- but I'm not sure I understand why we specifically mention the `HostRoot` type here, since we are ignoring everything other than `SuspenseComponent`.~~ Ah, I was missing the context of how `attemptToDispatchEvent()` worked.